executor/builder.go

```diff
@@ -1103,12 +1077,13 @@ func buildNoRangeIndexLookUpReader(b *executorBuilder, v *plan.PhysicalIndexLook
 		desc:              is.Desc,
 		tableRequest:      tableReq,
 		columns:           is.Columns,
-		handleCol:         handleCol,
 		priority:          b.priority,
-		tableReaderSchema: tableReaderSchema,
 		dataReaderBuilder: &dataReaderBuilder{executorBuilder: b},
 		batchSize:         128,
 	}
+	if cols, ok := v.Schema().TblID2Handle[is.Table.ID]; ok {
```

Comment:
If there is no handle column, `e.handleIndex` will be 0, will this do some bad effects?

Reply:
If there is no handle column, it indicates we needn't to sort.

Reply:
We'd better add this into the comment  of `handleIndex`. It's used only when `e.keepOrder` is true.

Reply:
fixed.